### ISSUE TITLE
Create baseline classifier using PredNet features

### ISSUE THREAD

**Issue by fonsc** [thefonseca/predictive-coding#7]
Baseline model should be CNN to take as input PredNet R3 features and predict activity labels.

Referecence: https://blog.keras.io/building-powerful-image-classification-models-using-very-little-data.html
